src/gluonts/model/trivial/constant.py

```diff
@@ -18,7 +18,10 @@
 from gluonts.dataset.field_names import FieldName
 from gluonts.dataset.util import forecast_start
 from gluonts.model.forecast import SampleForecast
-from gluonts.model.predictor import FallbackPredictor, RepresentablePredictor
+from gluonts.model.predictor import (
+    FallbackPredictor,
+    RepresentablePredictor,
+)
```

Comment:
Can we undo the changes?

The trailing `,` means that black will format these on multiple lines. If you remove it, black should format it on one line.